### PR TITLE
plural tickets

### PR DESCRIPTION
**Pull request by cliffordp**
[ET-954]

Before:

image

After:

image

[ET-954]: https://moderntribe.atlassian.net/browse/ET-954

### CODE REVIEW

File: src/admin-views/tribe-options-tickets.php [hunk 1]
```diff
@@ -60,7 +60,7 @@
 	],
 	'ticket-enabled-post-types' => [
 		'type'            => 'checkbox_list',
-		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
+		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_plural_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
```
Comment (tr1b0t) [inline src/admin-views/tribe-options-tickets.php:63]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 5980a5aa0c283bf7fb93afdf81929d47</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

File: src/admin-views/tribe-options-tickets.php (hunk 1 above)
Comment (juanfra) [inline src/admin-views/tribe-options-tickets.php:63]:
```suggestion
		// Translators: %s Tickets, plural.
		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_plural_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
```

**Review by juanfra — CHANGES_REQUESTED** [review]
Looking good! Let's add the translator comment and a readme entry.

File: src/admin-views/tribe-options-tickets.php (hunk 1 above)
Comment (cliffordp) [reply src/admin-views/tribe-options-tickets.php]: done

Comment (cliffordp) [reply src/admin-views/tribe-options-tickets.php]: I didn't think this required a changelog entry, but I added that as well.

File: src/admin-views/tribe-options-tickets.php [hunk 2]
```diff
@@ -60,7 +60,17 @@
 	],
 	'ticket-enabled-post-types' => [
 		'type'            => 'checkbox_list',
-		'label'           => esc_html( sprintf( _x( 'Post types that can have %s', 'tickets fields settings enabled post types', 'event-tickets' ), tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_enabled_post_types' ) ) ),
+		'label'           => esc_html(
+			sprintf(
+			// Translators: %s: dynamic "tickets" text.
```
Comment (sc0ttkclark) [inline src/admin-views/tribe-options-tickets.php:65]:
```suggestion
				// Translators: %s: dynamic "tickets" text.
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**